Makefile

```diff
@@ -23,6 +23,7 @@ endif # TARGET
 SERVICE := $(or ${SERVICE},quay.io/ocpmetal/assisted-service:latest)
 SERVICE_ONPREM := $(or ${SERVICE_ONPREM},quay.io/ocpmetal/assisted-service-onprem:latest)
 ISO_CREATION := $(or ${ISO_CREATION},quay.io/ocpmetal/assisted-iso-create:latest)
+OPENSHIFT_INSTALL_RELEASE_IMAGE := $(or ${OPENSHIFT_INSTALL_RELEASE_IMAGE},quay.io/ocpmetal/ocp-release:v1.0.9.2)
```

Comment:
I'm not sure we want to use "quay.io/ocpmetal/ocp-release:v1.0.9.2" as default.
There is a default openshift release image in the [code](https://github.com/openshift/assisted-service/blob/bc77fd1ede9267cdad76c01f8f0afe9097709497/pkg/job/job.go#L61).
And there is another default in the [config map](https://github.com/openshift/assisted-service/blob/8ac0e2d530fcb1313d76504b28790a8d6e184b76/deploy/assisted-service-configmap.yaml#L12)

I suggest checking which one we actually use in our CI and align them (preferably remove one of them).
Also don't think we should now have a default for the Makefile var as well.
If it's explicitly defined in the env we should pass it to tools/deploy_assisted_installer_configmap.py 
If not it has a [default](https://github.com/openshift/assisted-service/pull/443/files#diff-903ac12ee4b3b82ee5b606d4037d7c91R15).

Reply:
No worries, I will put again the original default when I finish solving the issues that I am facing :+1:

Reply:
I agree with @eranco74 
IMO We should remove the one from the code.

Reply:
The main default one was on the yaml file, the other one in Go code will ensure to check first the env var before setup the default one,

I've put the same value on the Makefile to have it set just in 1 place, I've set also in the same place at test-infra repository. Any suggestion for a right place (if you don't like the Makefile) to set the default variable @eranco74?

IMHO the Makefile should contain the default variables for (building, compile and deployment), if not you will force the contributors to swim into the py code with all the diferrent places, utils and functions to manipulate the values.